### PR TITLE
Profile page

### PR DESCRIPTION
**Pull request by alisondraV**

### CODE REVIEW

**Review by ANDREYDEN — CHANGES_REQUESTED** [review]
Awesome work!
1. When a user switches families (accepts the invite), there's a type error being thrown in the console. This might be fixed by using `CurrentFamily.listenForChanges` and forwarding the user back to the family page.
2. When updating the profile with a user without a family it fails with a family reference error.

File: src/views/profile-settings/Invitations.vue [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+  private async getInvitations() {
+    this.invitations = await Firestore.instance.getInvitations(this.user?.email ?? '');
+  }
+
+  async handleAcceptInvite(familyId: string) {
+    if (!this.user || !this.user.email) throw new Error('Unauthorized!');
+
+    try {
+      // TODO: update user's family
```
[the remark this answers is not in the document]
Comment (alisondraV) [reply src/views/profile-settings/Invitations.vue:66]: It was related to reloading the family. I added your favourite `CurrentFamily.instance.listenForChanges();` method. Seems to work😅

File: src/views/profile-settings/QuitFamily.vue [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  family: Family | null = null;
+  familyNameInputValue = '';
+
+  async mounted() {
+    this.family = await CurrentFamily.instance.getCurrentFamily();
+  }
+
+  async handleQuit() {
+    await CurrentFamily.instance.quit();
+    window.location.reload();
```
[the remark this answers is not in the document]
Comment (alisondraV) [reply src/views/profile-settings/QuitFamily.vue:47]: Opened up a new issue for this: #80

File: src/views/profile-settings/Invitations.vue [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+      await CurrentFamily.instance.switchTo(familyId, this.user.email);
+      await this.getInvitations();
+      this.isPositive = true;
+      await this.showAlert("You've accepted the invitation");
```
[the remark this answers is not in the document]
Comment (alisondraV) [reply src/views/profile-settings/Invitations.vue:70]: Opened up a new issue for this: #81

File: src/views/profile-settings/Family.vue [hunk 4]
```diff
@@ -103,17 +104,21 @@ export default class AppMain extends AlertMixin {
...
     await CurrentFamily.instance.listenForChanges(snapshot => {
       const family = snapshot.data() as Family;
       this.members = family?.members ?? [];
       this.pendingMembers = family?.pendingMembers ?? [];
       this.family = family;
     });
+
+    try {
+      // TODO: resolve CORS
+      this.familyMembers = await Firestore.instance.getUsersByEmail(this.members);
```
Comment (ANDREYDEN) [inline src/views/profile-settings/Family.vue:118]: `this.members` will be empty on start.
```suggestion
      this.familyMembers = await Firestore.instance.getUsersByEmail(this.family.members);
```

**Review by ANDREYDEN — CHANGES_REQUESTED** [review]
All good except that one thing.

**Review by ANDREYDEN — APPROVED** [review]
Ship it!

**Pull request merged**